sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -5553,9 +5553,9 @@ def test_issue_7180():
 
 
 def test_pretty_Complement():
-    assert pretty(S.Reals - S.Naturals) == '(-oo, oo) \ Naturals()'
+    assert pretty(S.Reals - S.Naturals) == 'Interval(-oo, oo, True, True) \ S.Naturals'
```

Comment:
@asmeurer @Upabjojr The test file has a redefinition of `pretty` [here](https://github.com/SagarB-97/sympy/blob/fc15454788a0f7af653740a96fb6dad81b469f8b/sympy/printing/pretty/tests/test_pretty.py#L210-L212).

So `use_unicode` is set to `False` and `S.Reals` is being printed using the default printer ie. `str`. In default printer, I can either make it print `S.Reals` (analogous to what is done for `S.Naturals`, `S.Integers`) or make it print `(-oo, oo)`. What would be best?

I will now assume it best to maintain uniformity and make changes.